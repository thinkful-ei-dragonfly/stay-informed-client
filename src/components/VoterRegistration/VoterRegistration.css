.reg-vote {
  display: flex;
  flex-direction: column;
}

.dash-link {
  /* margin: 40px; */
  font-size: 27px;
  color: black;
  margin-top: 30px;
  margin-bottom: 20px;
  width: 100%;
  background-color: rgb(180, 210, 230)
}