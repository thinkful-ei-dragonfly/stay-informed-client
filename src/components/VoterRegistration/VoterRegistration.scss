.reg-vote{
  position: relative;
  iframe{
    position: relative;
    z-index: 10;
  }
  .loading{
    position: absolute;
    z-index: 1;
    left: 0;
    right: 0;
    text-align: center;
  }
  .dash-link {
    /* margin: 40px; */
    font-size: 27px;
    color: black;
    margin-top: 30px;
    margin-bottom: 20px;
    width: 100%;
    background-color: rgb(180, 210, 230)
  }
}

