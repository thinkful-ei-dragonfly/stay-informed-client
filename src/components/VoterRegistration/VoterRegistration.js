import React, { Component } from 'react';
import { Link } from 'react-router-dom'
import './VoterRegistration.scss'

export default class VoterRegistration extends Component {
  render() {
    return (
      <div className="reg-vote">
       <Link to="/dashboard" className="dash-link">Navigate back to dashboard</Link>
       <p class="loading">Loading...</p>
        <iframe
          title="Use this form to register to vote in your local, state, and national elections."
          src="https://register.rockthevote.com/?partner=1&source=ovrpage"
          width="100%"
          height="1500"
          marginHeight="0"
          frameBorder="0"
        />
      </div>
    );
  }
}
