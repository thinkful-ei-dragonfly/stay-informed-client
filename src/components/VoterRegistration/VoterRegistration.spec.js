import React from 'react';
import { shallow } from 'enzyme'
import toJson from 'enzyme-to-json'
import VoterRegistration from './VoterRegistration'

describe(`VoterRegistration component`, () => {
  it('renders a form.VoterRegistration by default', () => {
    const wrapper = shallow(<VoterRegistration />)
    expect(toJson(wrapper)).toMatchSnapshot()
  })
})
