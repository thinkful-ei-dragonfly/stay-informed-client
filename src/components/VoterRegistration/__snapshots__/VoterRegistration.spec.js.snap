// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`VoterRegistration component renders a form.VoterRegistration by default 1`] = `
<div
  className="reg-vote"
>
  <Link
    className="dash-link"
    to="/dashboard"
  >
    Navigate back to dashboard
  </Link>
  <iframe
    frameBorder="0"
    height="1500"
    marginHeight="0"
    src="https://register.rockthevote.com/?partner=1&source=ovrpage"
    title="Use this form to register to vote in your local, state, and national elections."
    width="100%"
  />
</div>
`;
